Source/HelixToolkit.Wpf/Helpers/MeshBuilder.cs

```diff
@@ -1966,10 +1966,10 @@ public void AddPolygon(IList<int> vertexIndices)
         }
 
         /// <summary>
-        /// Adds a polygon defined by vertex indices (uses the cutting ears algorithm).
+        /// Adds a polygon defined by vertex indices (uses the sweep line algorithm).
         /// </summary>
         /// <param name="vertexIndices">The vertex indices.</param>
-        public void AddPolygonByCuttingEars(IList<int> vertexIndices)
+        public void AddPolygonBySweepLine(IList<int> vertexIndices)

```

Comment:
I think the method name should be `AddPolygon` and the `AddPolygonByCuttingEars` should be marked obsolete